### PR TITLE
[Kodi] Feature Kodi currenttime & duration

### PR DESCRIPTION
**Pull request by cweitkamp**
- Added channels for current time and duration
- Renamed enum constants to use capital letters (addressed https://github.com/openhab/openhab2-addons/pull/3479#discussion_r183437757) 

Feature request of the community (see https://community.openhab.org/t/kodi-inputaction/40276/10?u=cweitkamp)

Testversion: [org.openhab.binding.kodi-2.3.0-SNAPSHOT.jar.zip](https://github.com/openhab/openhab2-addons/files/1958673/org.openhab.binding.kodi-2.3.0-SNAPSHOT.jar.zip)

Signed-off-by: Christoph Weitkamp <github@christophweitkamp.de>

### CODE REVIEW

File: addons/binding/org.openhab.binding.kodi/src/main/java/org/openhab/binding/kodi/internal/KodiEventListener.java [hunk 1]
```diff
@@ -23,12 +23,12 @@
...
-        End,
-        Stop,
-        Rewind,
-        FastForward
+        PLAY,
+        PAUSE,
+        END,
+        STOP,
+        REWIND,
+        FASTFORWARD
```
Comment (martinvw) [inline addons/binding/org.openhab.binding.kodi/src/main/java/org/openhab/binding/kodi/internal/KodiEventListener.java]: FASTFORWARD or FAST_FORWARD

**Comment by martinvw** [comment]
@cweitkamp can you resolve the conflicts?

File: addons/binding/org.openhab.binding.kodi/src/main/java/org/openhab/binding/kodi/internal/KodiEventListener.java (hunk 1 above)
Comment (cweitkamp) [reply addons/binding/org.openhab.binding.kodi/src/main/java/org/openhab/binding/kodi/internal/KodiEventListener.java]: FASTFORWARD

**Comment by cweitkamp** [comment]
@martinvw Done. I rebased to current master.

**Pull request merged**